cli/flox-rust-sdk/src/models/environment/core_environment.rs

```diff
@@ -742,6 +781,7 @@ mod tests {
 
     /// A no-op with edit returns EditResult::Unchanged
     #[test]
+    #[serial]
```

Comment:
Question: Is this `serial` necessary now because all of the tests using `flox_instance_with_global_lock` would be racing to see which one creates it first?

Reply:
I just have it on tests that invoke pkgdb at the moment, even if a scrape isn't triggered. It's possible we can take them all off, but last time I asked @billlevine he wasn't certain we won't have a race even when the input has already been scraped.